addons/calendar/models/calendar_recurrence.py

```diff
@@ -133,10 +133,9 @@ def _compute_name(self):
             else:
                 end = ''
 
-            if recurrence.rrule_type == 'weeky':
+            if recurrence.rrule_type == 'weekly':
                 weekdays = recurrence._get_week_days()
-                weekday_fields = (self._fields[weekday_to_field(w)] for w in weekdays)
-                on = _("on %s,") % ", ".join([field.string for field in weekday_fields])
+                on = _("on %s,") % ", ".join(self._fields[str(w).lower()].string for w in weekdays)
```

Comment:
as we are on the suject, `self._fields[...].string` is the raw untranslated value, you probably want to use `fields_get()` instead